### DISCUSSION TITLE
How to compare two folders that contain files in different codepages

### DISCUSSION THREAD

**Discussion by rchelaru23** [WinMerge/winmerge#698] [Q&A]
Good day. 

I am trying to compare two folders that contain multiple files and subfolders, each folder contains the files in a single codepage. 

Lets say, folder1 would be UTF-8 and folder2 would be the **same files** but in ISO-8859-1. 

Currently when trying to compare the two, I would get differences in some files that contain characters unavailable in the others codepage. I have managed to compare two files in different codepages by selecting the codepage and "Affects" left or right side, but given the number of files in the two folders this would take quite long to do individually for each file.

Is there any way to compare the two folders using different codepages?

Thank you

**Comment by sdottaka** [comment]
Unfortunately, WinMerge cannot explicitly specify the codepage of files when comparing folders.
WinMerge automatically detects the code page of the file, but do you say that WinMerge falsely detected the code page of the files?

If it is annoying to set each file in the file encoding dialog, you can select multiple files in the folder comparison window and specify the file code page as shown below. (This feature didn't work in 2.16.10 and has been fixed in 2.16.11 beta.)

![image](https://user-images.githubusercontent.com/98126/112559777-0b4d3a00-8e15-11eb-9cc7-7cbe3eeeaa55.png)

However, unfortunately, when re-comparing the folders on the specified code page, the code page is restored.
